docker-compose.yml

```diff
@@ -1,38 +1,37 @@
-version: '3'
+version: '3.7'
 services:
-  
+
   app:
     build:
       context: .
       dockerfile: Dockerfile.dev
-    env_file: 
+    env_file:
       - .env
     environment:
-      - FLASK_APP=app.factory:create_app('development')
-      - DATABASE_URL=postgresql://supportService:supportService@db/supportService
       - FLASK_ENV=development
+      - FLASK_APP=app.factory:create_app('development')
       - REDIS_HOST=cache
+      - LOG_TO_STDOUT=1
     volumes:
-      - .:/usr/src/app
+      - .:/app
     ports:
-      - "5000:5000"
-    command: ["./scripts/start_dev.sh", "5000"]
+      - 3000:3000
+    command: flask run -h 0.0.0.0 -p 3000 --reload
     depends_on:
-      - db
       - cache
 
-  db:
```

Comment:
@rmanalan is there a specific reason why you got rid of the database here?

Reply:
Because I realized that the dev env doesn't require PG and will default to sqlite.